profile/CSIP.xml

```diff
@@ -9,7 +9,7 @@
 <METS_Profile xmlns="http://www.loc.gov/METS_Profile/v2"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xmlns:mets="http://www.loc.gov/METS/"
-    xmlns:csip="DILCIS"
+    xmlns:csip="https://DILCIS.eu/XML/METS/CSIPExtensionMETS"
```

Comment:
URLs aren't case sensistive and it's unusual to express case, I think that these should be lower case.